### PR TITLE
[mssql] Fix module not found error caused by pyodbc (#366)

### PR DESCRIPTION
**Pull request by yesidpol**
If pyodbc is not installed (exception ModuleNotFoundError), SqlServer is not added to the list of available database

### CODE REVIEW

File: QgisModelBaker/libqgsprojectgen/db_factory/db_simple_factory.py [hunk 1]
```diff
@@ -17,19 +17,30 @@
  ***************************************************************************/
 """
 from .db_factory import DbFactory
-from .pg_factory import PgFactory
-from .gpkg_factory import GpkgFactory
-from .mssql_factory import MssqlFactory
 from QgisModelBaker.libili2db.globals import DbIliMode
 
+available_databases = dict()
```
Comment (m-kuhn) [inline QgisModelBaker/libqgsprojectgen/db_factory/db_simple_factory.py]:
```suggestion
available_database_factories = dict()
```

I think that would make line 58 a bit easier to read

**Comment by m-kuhn** [comment]
Could you make another pull request against the ili2pg4 branch?

File: QgisModelBaker/libqgsprojectgen/db_factory/db_simple_factory.py (hunk 1 above)
Comment (yesidpol) [reply QgisModelBaker/libqgsprojectgen/db_factory/db_simple_factory.py]: Done!

**Comment by yesidpol** [comment]
> Could you make another pull request against the ili2pg4 branch?


> Could you make another pull request against the ili2pg4 branch?

Ok

**Pull request merged**